modules/ephemeral-storage-integration-between-shared-resources-insights-operator-and-openshift-builds.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: REFERENCE
+
+[id="ephemeral-storage-integration-between-shared-resources-insights-operator-and-openshift-builds_{context}"]
+= Integration between shared resources, Insights Operator, and OpenShift Builds
+
+[role="_abstract"]
+Integration between shared resources, Insights Operator, and OpenShift Builds makes using Red Hat subscriptions (RHEL entitlements) easier.
+
+Previously, in {product-title} 4.9.x and earlier, you manually imported your credentials or copied them to each project or namespace where you were running builds.
+
+Now, in {product-title} 4.10 and later, OpenShift Builds can employ Red Hat subscriptions (RHEL entitlements) by using shared resources and the simple content access feature provided by Insights Operator:
```

Comment:
```suggestion
Now, in {product-title} 4.10 and later, OpenShift Builds can employ Red Hat subscriptions (RHEL entitlements) by using Shared Resources and the Simple Content Access feature provided by Insights Operator:
```

Reply:
also, add a link to the 4.10 version of https://docs.openshift.com/container-platform/4.9/support/remote_health_monitoring/insights-operator-simple-access.html

Reply:
The terms "shared resources" and "simple content access" are sentence case throughout the OpenShift docs.

Reply:
Added the following cross-references to the `storage/container_storage_interface/ephemeral-storage-shared-resource-csi-driver-operator.adoc` file:
```
.Additional resources

xref:../../support/remote_health_monitoring/insights-operator-simple-access.adoc#insights-operator-simple-access[Importing simple content access certificates with Insights Operator]
xref:../../cicd/builds/running-entitled-builds.html#builds-source-secrets-entitlements_running-entitled-builds[Adding subscription entitlements as a build secret]
```
To readers, these cross-references look like they're at the bottom of the `modules/ephemeral-storage-integration-between-shared-resources-insights-operator-and-openshift-builds.adoc` file.

I also added "See the links in the following "Additional resources" section." to one of the bullets in the  the `modules/ephemeral-storage-integration-between-shared-resources-insights-operator-and-openshift-builds.adoc` file.

All of this is because of a mind-boggling limitation that prevents us from using xrefs in module files.